pkg/amber/provenance.go

```diff
@@ -70,6 +70,25 @@ func validateSLSAProvenanceJSON(provenanceFile []byte) error {
 	return nil
 }
 
+// GetConfigSourceURI returns the URI of the configuration.
+func GetConfigSourceURI(pred *slsav02.ProvenancePredicate) string {
```

Comment:
Can you add tests for this and `GetMaterialsGitURI`? The tests in `verifier_test` do not cover these.

Reply:
I would add tests in #198 (and modify `provenance.json`). This is then be covered by `TestGenerateEndorsement_SingleValidEndorsement`.

Reply:
Great. Thanks.